app/src/main/java/deakin/gopher/guardian/view/general/PatientListActivity.java

```diff
@@ -20,116 +20,116 @@
 import deakin.gopher.guardian.view.patient.careplan.CarePlanActivity;
 
 public class PatientListActivity extends AppCompatActivity {
-  RecyclerView patient_list_recyclerView;
-  PatientListAdapter patientListAdapter;
-  Query query;
-  CardView overview_cardview;
-  SearchView patient_searchView;
+    RecyclerView patient_list_recyclerView;
+    PatientListAdapter patientListAdapter;
+    Query query;
+    CardView overview_cardview;
+    SearchView patient_searchView;
 
-  @Override
-  protected void onCreate(final Bundle savedInstanceState) {
```

Comment:
What's going on with all these formatting changes. I don't think this is necessary?